backend/pokemon/helpers.py

```diff
@@ -13,5 +13,14 @@ def get_pokemon_from_api(poke_id):
         "hp": data["stats"][0]["base_stat"],
         "name": data["name"],
         "img_url": data["sprites"]["front_default"],
+        "pokemon_id": data["id"]
     }
     return info
+
+
+def verify_pokemon_exists_api(pokemons):
```

Comment:
those pokemons are pokemon names actually, right? worth making this clearer. 
consider change the variable name to `pokemon_names`

Reply:
Great! I'm going to change this :D